TESTS/mbedtls/multi/main.cpp

```diff
@@ -23,6 +23,17 @@
 
 #include "mbedtls/sha256.h"
 
+#if defined(MBEDTLS_PLATFORM_C)
+#include "mbedtls/platform.h"
+#else
+#include <stdio.h>
+#include <stdlib.h>
+#define mbedtls_printf     printf
+#define mbedtls_snprintf   snprintf
```

Comment:
It looks as if we only need `mbedtls_printf` in this file, so can we get rid of `stdlib.h`, `mbedtls_snprintf`, `mbedtls_exit`, `MBEDTLS_EXIT_SUCCESS` and `MBEDTLS_EXIT_FAILURE`?

Reply:
Ok, I'll update the PR.